README.md

```diff
@@ -252,5 +252,17 @@ https://local.broadinstitute.org:50443/status](https://local.broadinstitute.org:
 [Swagger page:
 https://local.broadinstitute.org:50443/#/](https://local.broadinstitute.org:50443/#/)
 
+#### Verify that Sam is running anywhere
+Sam includes smoke tests that can be run anywhere and test a running Sam instance running anywhere.  These tests run
+quickly and in 2 modes: authenticated and unauthenticated.
```

Comment:
Yeah, 100% think we should make the `/status` endpoint better but I think that's also just another ticket.  I don't disagree that it would be great if the `/status` endpoint just tells us everything, but the reality is, sometimes, we still don't trust it and go in and manually poke at things.  These tests are another alternative that should make the "go click around" test trivially easy.